### PR TITLE
feat(SnapCard): new component

### PR DESCRIPTION
**Pull request by atabel**
jira: WEB-446 

<del>Pending definition, @AnaMontes11 @yceballost:</del>
- [x] <del>Text alignment</del>
- [x] <del>Slot</del>
- [x] <del>Inverse</del>

### CODE REVIEW

**Review by yceballost — CHANGES_REQUESTED** [review]

File: flow-defs/card.js.flow [hunk 1]
```diff
@@ -41,4 +41,33 @@ declare type DataCardProps = {
   "aria-label"?: string,
 };
 declare export var DataCard: React.AbstractComponent<DataCardProps, HTMLDivElement>;
+declare type SnapCardBaseProps = {
+  icon?: React.Element<any>,
```
[the remark this answers is not in the document]
Comment (atabel) [reply flow-defs/card.js.flow:45]: I used the same prop name we use in other card components

**Review by RubenMorPov — APPROVED** [review]

File: flow-defs/card.js.flow (hunk 1 above)
[the remark this answers is not in the document]
Comment (yceballost) [reply flow-defs/card.js.flow:45]: but it is quite weird use 🤔

```
icon={
        <Image
          src="https://i.imgur.com/aEVtKsE.jpg"
          height={64}
          aspectRatio="16:9"
        />
      }
```

if you think is ok, you can solve this comment

[the remark this answers is not in the document]
Comment (atabel) [reply flow-defs/card.js.flow:45]: I prefer to be consistent in components api

**Review by yceballost — APPROVED** [review]

File: src/card.tsx [hunk 2]
```diff
@@ -216,3 +217,113 @@ export const DataCard = React.forwardRef<HTMLDivElement, DataCardProps>(
         );
     }
 );
+
+const useSnapCardStyles = createUseStyles((theme) => ({
```
Comment (pladaria) [inline src/card.tsx:221]: why don't define this component in its own file? (snap-card.tsx)

**Review by pladaria — APPROVED** [review]

**Review by aweell — APPROVED** [review]

**Review by RubenMorPov — APPROVED** [review]

File: src/card.tsx [hunk 3]
```diff
@@ -216,3 +217,113 @@ export const DataCard = React.forwardRef<HTMLDivElement, DataCardProps>(
...
+        },
+        ref
+    ) => {
+        const isTouchable = Boolean(touchableProps.to || touchableProps.href || touchableProps.onPress);
+        const classes = useSnapCardStyles({isTouchable, isInverse, hasIcon: !!icon});
+        const {colors} = useTheme();
+
+        return (
+            <Boxed className={classes.boxed} dataAttributes={dataAttributes} ref={ref} isInverse={isInverse}>
+                <Touchable maybe {...touchableProps} className={classes.touchable} aria-label={ariaLabel}>
```
[the remark this answers is not in the document]
Comment (atabel) [reply src/card.tsx:306]: I don't think is a good idea. If no ariaLabel is provided, screen readers will read the text inside the button/a as the label: the title and subtitle.

**Pull request merged**